### PR TITLE
Update introduction.tex

### PR DESCRIPTION
**Pull request by dgarcia360**
Added #12 text suggestions from @btinsman

### CODE REVIEW

File: whitepaper/chapters/introduction.tex [hunk 1]
```diff
@@ -15,10 +15,12 @@ \section{Introduction}
 These choices necessarily sacrifice some throughput relative to other protocols, but they seem most consistent with the philosophical underpinnings of Bitcoin.
 
 As part of a focus on trustlessness, the following features were added relative to NEM:
-- Block headers can be synced without transaction data, while allowing verification of chain integrity
-- Transaction merkle trees allow cryptographic proofs of transaction containment (or not) within blocks
-- Receipts increase the transparency of indirectly triggered state changes
-- State proofs allow trustless verification of specific state within the blockchain
+\begin{itemize}
+\item Block headers can be synced without transaction data, while allowing verification of chain integrity.
```
Comment (Jaguar0625) [inline whitepaper/chapters/introduction.tex]: i think @gimer said that items should be wrapped in braces {}

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
one comment, otherwise looks good

File: whitepaper/chapters/introduction.tex [hunk 2]
```diff
@@ -15,22 +15,24 @@ \section{Introduction}
 These choices necessarily sacrifice some throughput relative to other protocols, but they seem most consistent with the philosophical underpinnings of Bitcoin.
 
 As part of a focus on trustlessness, the following features were added relative to NEM:
-- Block headers can be synced without transaction data, while allowing verification of chain integrity
-- Transaction merkle trees allow cryptographic proofs of transaction containment (or not) within blocks
-- Receipts increase the transparency of indirectly triggered state changes
-- State proofs allow trustless verification of specific state within the blockchain
+\begin{itemize}
+  \item {Block headers can be synced without transaction data, while allowing verification of chain integrity.}
```
Comment (Jaguar0625) [inline whitepaper/chapters/introduction.tex]: please indent with TABs

Comment (Jaguar0625) [reply whitepaper/chapters/introduction.tex]: no space after item

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

**Comment by dgarcia360** [comment]
@Jaguar0625 should be ok now

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**